### PR TITLE
[Docs site] Added how-to structured data options + updated nameserver page for a test

### PR DESCRIPTION
**Pull request by kodster28**
PCX-5724

Preview passed validation at:
- https://search.google.com/test/rich-results/result/r%2Fhow-to?id=4HHiW_7poXs6KQeRRs5eQQ
- https://validator.schema.org/#url=https%3A%2F%2Fhow-to-structured-data.cloudflare-docs-7ou.pages.dev%2Fdns%2Fzone-setups%2Ffull-setup%2Fsetup%2F

There's a small issue in the preview where they're saying the `open external link` info might show up... but this wasn't the case when we pushed the FAQs live, so think it's likely safe.

### CODE REVIEW

File: layouts/_default/_markup/render-heading.html [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {{- $ID := (.Anchor | safeURL) -}}
 {{- $text := (.Text | safeHTML) -}}
 {{- if eq .Level 1 }}
-  <h1 id="{{ $ID }}">{{ $text }}</h1>
+  {{- $howToSchema := "" }}
```
Comment (pedrosousa) [inline layouts/_default/_markup/render-heading.html]: Not used?

File: layouts/shortcodes/how-to-section.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{{- $headingLevel := (default 2 (.Get "headingLevel")) | string -}}
```
Comment (pedrosousa) [inline layouts/shortcodes/how-to-section.html:1]: I guess we're planning to use this `how-to-section` shortcode later.

File: layouts/shortcodes/how-to-step.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{{- $title := (default "" (.Get "title")) -}}
+{{- $ID := $title | urlize -}}
+
+{{- with .Get "insideSection" -}}
```
Comment (pedrosousa) [inline layouts/shortcodes/how-to-step.html:4]: I suppose this parameter would be used inside a `how-to-section` shortcode (right now it's not being used).

File: content/dns/zone-setups/full-setup/setup.md [hunk 4]
```diff
@@ -4,13 +4,22 @@ pcx_content_type: tutorial
 weight: 1
 meta:
   title: Change your authoritative nameservers (Full setup)
+structured_data: true
 ---
 
 # Change your authoritative nameservers (Full setup)
 
 {{<render file="_full-setup-definition.md">}}
```
Comment (pedrosousa) [inline content/dns/zone-setups/full-setup/setup.md:12]: Feels a bit odd to lose this intro in structured data, but I couldn't find a proper attribute for it.

File: layouts/shortcodes/how-to-step.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  <div itemprop="itemListElement" itemscope itemtype="https://schema.org/HowToStep" class="howToStep">
+{{- else -}}
+  <div itemprop="step" itemscope itemtype="https://schema.org/HowToStep" class="howToSection">
```
Comment (pedrosousa) [inline layouts/shortcodes/how-to-step.html]: Not sure about the `howToSection` class here when there's no parent section and this is a step (I guess it depends on the styling we wish to have for sections), but let's get back to this later when we start having sections.

**Review by pedrosousa — APPROVED** [review]

File: content/dns/zone-setups/full-setup/setup.md (hunk 4 above)
Comment (kodster28) [reply content/dns/zone-setups/full-setup/setup.md:12]: Yeah... I think they pull that in via the normal metadata description, but agreed that it's sorta strange.

Seems to show up in the preview though.

**Pull request merged**